jest.config.ts

```diff
@@ -0,0 +1,42 @@
+import type {Config} from '@jest/types';
+
+const config: Config.InitialOptions = {
+  verbose: true,
+  preset: 'ts-jest',
+  coverageDirectory: '__coverage__',
+  collectCoverage: true,
+  transformIgnorePatterns: [
+    'node_modules/(?!@patternfly|lodash-es|@console|@novnc|@spice-project|@popperjs|i18next)'
+  ],
+  coverageReporters: [
+    'json',
+    'lcov',
+    'text',
+    'text-summary'
+  ],
+  collectCoverageFrom: [
+    'packages/*/src/**/*.{js,jsx,ts,tsx}',
+    '!**/node_modules/**',
+    '!**/dist/**'
+  ],
+  testURL: 'http://localhost',
+  testRegex: '.*\\.(spec|test)\\.(ts|tsx|js|jsx)$',
+  testEnvironment: 'jsdom',
+  testPathIgnorePatterns: [
+    '<rootDir>/node_modules/'
+  ],
+  roots: ['<rootDir>/packages/'],
+  globalSetup: '<rootDir>/config/globalSetup.ts',
+  setupFilesAfterEnv: ['<rootDir>/config/setupTests.ts'],
+  moduleNameMapper: {
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|css|less|scss)$': 'identity-obj-proxy',
+    'lodash-es': 'lodash'
```

Comment:
I don't think we need this. We aim to use `lodash-es` (only) for all of our packages.

Reply:
In all of built packages the `lodash-es` is used, however since jest doesn't understand new ES modules we have to provide a fallback to `lodash` we would either have to pull in babel config, or use this moduleMapper.

Reply:
We're already un-ignoring `lodash-es` from `ts-node` processing, so Jest should be able to process `lodash-es` imports just fine (?)